pr_agent/algo/pr_processing.py

```diff
@@ -30,15 +32,17 @@ def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHa
     pr_languages = sort_files_by_main_languages(git_provider.get_languages(), git_provider.pr.diff_files)
 
     # generate a standard diff string, with patch extension
-    patches_extended, total_tokens = pr_generate_extended_diff(pr_languages, token_handler)
+    patches_extended, total_tokens = pr_generate_extended_diff(pr_languages, token_handler,
+                                                               add_line_numbers_to_hunks)
```

Comment:
**Suggestion:** Consider using a more descriptive name for the boolean parameter 'add_line_numbers_to_hunks' in the function 'get_pr_diff'. The current name does not clearly indicate what the parameter does.
```suggestion
    def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHandler,
                    should_add_line_numbers_to_hunks: bool = False) -> str:
```